__tests__/get.test.js

```diff
@@ -85,55 +84,106 @@ describe("GET /api/reviews", () => {
-            title: expect.any(String),
-            review_id: 3,
-            category: expect.any(String),
-            review_img_url: expect.any(String),
-            review_body: expect.any(String),
-            created_at: expect.any(String),
-            votes: expect.any(Number),
-            designer: expect.any(String),
-            comment_count: expect.any(String),
-          });
+          expect(result.body).toEqual([]);
         });
     });
-    test("order", () => {
-      return request(app)
-        .get("/api/reviews?order=desc")
-        .expect(200)
-        .then((result) => {
-          expect(result.body.reviews).toBeInstanceOf(Array);
-          expect(result.body.reviews.length).toBeGreaterThan(0);
-          expect(result.body.reviews[0]).toMatchObject({
-            owner: expect.any(String),
-            title: expect.any(String),
-            review_id: 13,
-            category: expect.any(String),
-            review_img_url: expect.any(String),
-            review_body: expect.any(String),
-            created_at: expect.any(String),
-            votes: expect.any(Number),
-            designer: expect.any(String),
-            comment_count: expect.any(String),
+
+    describe("sort_by", () => {
+      test("shoudl default to created_at", () => {
+        return request(app)
+          .get("/api/reviews?sort_by=default")
+          .expect(200)
+          .then((result) => {
+            expect(result.body.reviews.length).toBeGreaterThan(0);
+            expect(result.body.reviews[0]).toMatchObject({
+              owner: expect.any(String),
+              title: expect.any(String),
+              review_id: 13,
+              category: expect.any(String),
+              review_img_url: expect.any(String),
+              review_body: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              designer: expect.any(String),
+              comment_count: expect.any(String),
+            });
           });
-        });
+      });
+      test("sort_by", () => {
+        return request(app)
+          .get("/api/reviews?sort_by=owner")
+          .expect(200)
+          .then((result) => {
+            expect(result.body.reviews.length).toBeGreaterThan(0);
+            expect(result.body.reviews[0]).toMatchObject({
+              owner: "bainesface",
+              title: expect.any(String),
+              review_id: 3,
+              category: expect.any(String),
+              review_img_url: expect.any(String),
+              review_body: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              designer: expect.any(String),
+              comment_count: expect.any(String),
+            });
+          });
+      });
+    });
+    describe("order", () => {
+      test("default order", () => {
+        return request(app)
+          .get("/api/reviews?order=desc")
```

Comment:
if this is the default order you should test that it's in this order without the query.